### DISCUSSION TITLE
Tool used to generate png images for Quarkus Guides?

### DISCUSSION THREAD

**Discussion by cerasmus-booking** [quarkusio/quarkus#24144] [Q&A] [answered]
Hi, does anyone know which tool is used to create the diagram PNG images on the Quarkus Guides site? For example, here is the Reactive Messaging Architecture https://quarkus.io/guides/kafka-reactive-getting-started#architecture.

(I couldn't find a discussion group on Quarkusio (project for the guides) so apologies if I need to ask somewhere more appropriate)

**Comment by maxandersen** [comment, marked as answer]
The tool is called @cescoffier - Clement is quite the artist and he has the "hobby" to make these diagram drawn by hand (on his ipad I believe).

**Reply by cerasmus-booking** [reply]
Haha, I see. Is he open source by any chance? :-)

Thx
